.github/workflows/showcase-test.yml

```diff
@@ -0,0 +1,33 @@
+name: showcase-tests
+run-name: Showcase Tests
+on:
+  push:
+    branches: [ master ]
+  pull-request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 7 * * *'  # 7am UTC every day
+
+jobs:
+  test:
+    name: Showcase Compile Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup Java JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'oracle'
+          cache: 'maven'
```

Comment:
will this be equivalent to using `actions/cache` like in https://github.com/finos/legend-engine/blob/master/.github/workflows/build.yml ?

Reply:
setup-java uses actions/cache under the hood: https://github.com/actions/setup-java#caching-packages-dependencies

Looking at that file, I believe the simple usage should be equivalent.